framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1699,6 +1702,27 @@ public AnnotatedDeclaredType getSelfType(Tree tree) {
     }
 
     /**
+     * Get the AnnotatedDeclaredType for a classTree without kicking off dataflow or calling
```

Comment:
Isn't this the exact same sequence of calls as before?
How is dataflow not kicked off?